snippets/Google.Logging.Log4Net.Snippets/project.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+  "version": "1.0.0-*",
```

Comment:
I'd get rid of most of this - it's possible that I haven't trimmed the other snippet projects as much as I should do, but basically we don't need much because we're not publishing the package.